Portal/View/UIKit/Renderers/TextFieldRenderer.swift

```diff
@@ -29,30 +30,32 @@ internal struct TextFieldRenderer<MessageType, RouteType: Route>: UIKitRenderer
         textField.removeTarget(.none, action: .none, for: .editingDidBegin)
         textField.removeTarget(.none, action: .none, for: .editingChanged)
         textField.removeTarget(.none, action: .none, for: .editingDidEnd)
+        if properties.shouldReturn {
+            textField.delegate = TextFieldDelegate.shared
```

Comment:
Delegate have weak references so what about making `UITextField` its own delegate by confirming to the protocol.

Reply:
You mean with an extension of UITextField?

Reply:
Yes